src/vivarium_nih_us_cvd/components/treatment.py

```diff
@@ -644,6 +659,55 @@ def apply_ldlc_treatment_ramp(
             + 1
         ).map(self.ldlc_treatment_map)
 
+        # Apply outreach intervention to groups 'newly_prescribed' (d, e, f)
+        # and simulants already on medication
+        maybe_enroll = newly_prescribed.union(currently_medicated)
+        pop_visitors = self.enroll_in_outreach(pop_visitors, maybe_enroll)
```

Comment:
This shouldn't happen in the treatment ramp functions themselves. You should just mark these simulants to be maybe enrolled and then call `enroll_in_outreach` once outside the ramp functions. As it is, if a simulant is on both types of medication this function will be called on them twice in the same timestep.